Source/mpq/mpq_writer.hpp

```diff
@@ -56,11 +64,11 @@ class MpqWriter {
 
 	LoggedFStream stream_;
 	std::string name_;
-	std::uintmax_t size_;
-	bool modified_;
-	bool exists_;
-	MpqHashEntry *hashTable_;
-	MpqBlockEntry *blockTable_;
+	std::uintmax_t size_ {};
+	bool modified_ {};

```

Comment:
You can also get rid of `modified` and treat it as `true`. Every time a file is opened, some changes are invariably done by the time it is closed.

Reply:
Leftover. Thanks 🙂